src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -174,6 +174,10 @@
             <tag name="console.command" command="translation:update" />
         </service>
 
+        <service id="console.command.uid_generate" class="Symfony\Bundle\FrameworkBundle\Command\UidGenerateCommand">
+            <tag name="console.command" command="uid:generate" />
+        </service>
```

Comment:
I'd only wire this command if the component is available.

Reply:
Yes, i've updated my code to do so. I've followed the way it's done for other commands: i still set the command in `console.xml`, but i remove the definition in the `FrameworkExtension.php` file if the Uid component is not installed (ie: if `AbstractUid` class doesn't exist).